draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3359,7 +3304,7 @@ implementations.
           |D=$L-PRIV-1   |              |              |
           |MA=$NAT-PUB-1 |              |              |
           |<-------------|              |              |
-          |Data flows    |              |              |
+          |              |              |              |
```

Comment:
"Data Flows" is still accurate here for the majority of use cases, as the pair can be used as soon as it becomes "valid," and most implementations will be attempting to minimize setup delay.

Reply:
Seems like it got dropped for some reason. I'll put it back.